smithy-diff/src/main/java/software/amazon/smithy/diff/evaluators/ModifiedTrait.java

```diff
@@ -294,8 +294,8 @@ List<ValidationEvent> validate(
                     return Collections.emptyList();
                 }
 
-                String leftPretty = Node.prettyPrintJson(left.toNode());
-                String rightPretty = Node.prettyPrintJson(right.toNode());
+                String leftPretty = Node.prettyPrintJsonWithBackticks(left.toNode());
```

Comment:
don't need .toNode on left or right. I know it was there before, but this is a good time to fix it.